### PR TITLE
Add option to disable the wireless interface when connected to prevent audio dropouts

### PR DESCRIPTION
**Pull request by c-larsen**
Added an option for the user to have the daemon automatically disable the wireless interface (default WLAN0) when a device is connected and enable it again when disconnected. This is to prevent the known problems with audio stuttering and dropouts when streaming audio via bluetooth while the WLAN interface is active (https://github.com/lukasjapan/bt-speaker/issues/4)

### CODE REVIEW

**Comment by Sphinxila** [comment]
Hi larsen, you can not merge your branch cause after merging the 'install.sh' download url will be changed to your github repository.
.

**Comment by c-larsen** [comment]
No, the github URL has been set back to the main repository as part of this commit https://github.com/lukasjapan/bt-speaker/pull/20/commits/aa474a1b3262d4b8f324d5f7c1727f1ce22ab06b

File: install.sh [hunk 1]
```diff
@@ -28,6 +20,19 @@ else
 fi
 echo "done."
 
+if [ $1 != "root" ]; then
```
Comment (pitpit) [inline install.sh]: Is this ability really needed to turn off wlan ?

**Comment by lukasjapan** [comment]
I think this kind of feature looks better inside the `connect_command `, `disconnect_command` hooks rather than the python code.

**Comment by djh816** [comment]
Just our of curiosity, what errors are you seeing that indicate wifi is causing dropouts? The audio pipe fix seems to fix any and all dropouts I was having and I have wifi enabled at all times still

**Comment by lukasjapan** [comment]
possibility to add this given by #37

not testet yet, additional permissions for btspeaker user may be needed, in worst case sudoers will do

**Pull request closed without merging**